### DISCUSSION TITLE
Difference accessing vault when offline vs deleted user?

### DISCUSSION THREAD

**Discussion by github4soft** [dani-garcia/vaultwarden#3153] [General]
Hi,

we're close to implement Vaultwarden in our company.
Because passwords are important (no surprise) we've tested what happens when the mobile client is offline. Access to the passwords/vault(?) is still possible but you can't change anything. Is the vault cached on my mobile client?
For long can I access my passwords/vault(?) without connection to the vault server?
When I delete a Vaultwarden user account, will access be possible, if the user is smart (and fast) enough to disable web-/network-access?

Thanks a lot for helping.

Kind regards

**Comment by kpfleming** [comment]
The vault is 'cached' on mobile clients the same way that it is in the browser extensions and desktop client.

The behavior of the clients is under control of Bitwarden, not Vaultwarden. If you want the ability to disable caching, for example, Bitwarden would need to implement that (as well as implementing server-side policies which push out to the clients to enforce it).

**Comment by github4soft** [comment]
Thanks a lot for your answer kpfleming.

I'll contact Bitwarden if there are any plans on their roadmap.
